Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANS/SANSSave.py

```diff
@@ -50,54 +69,49 @@ def PyInit(self):
             "Nexus",
             False,
             direction=Direction.Input,
-            doc="Save as nexus format. "
-            "Note that if file formats of the same type, e.g. .xml are chosen, then the "
-            "file format is appended to the file name.",
+            doc="Save as nexus format. " + FILE_FORMAT_DOC_SUFFIX,
         )
         self.declareProperty(
             "CanSAS",
             False,
             direction=Direction.Input,
-            doc="Save as CanSAS xml format."
-            "Note that if file formats of the same type, e.g. .xml are chosen, then the "
-            "file format is appended to the file name.",
+            doc="Save as CanSAS xml format. " + FILE_FORMAT_DOC_SUFFIX,
         )
         self.declareProperty(
             "NXcanSAS",
             False,
             direction=Direction.Input,
-            doc="Save as NXcanSAS format."
-            "Note that if file formats of the same type, e.g. .xml are chosen, then the "
-            "file format is appended to the file name.",
+            doc="Save as NXcanSAS format. " + FILE_FORMAT_DOC_SUFFIX,
+        )
+        self.declareProperty(
+            "PolarizedNXcanSAS",
+            False,
+            direction=Direction.Input,
+            doc="Save in PolarizedNXcanSAS format. " + FILE_FORMAT_DOC_SUFFIX,
         )
         self.declareProperty(
             "NistQxy",
             False,
             direction=Direction.Input,
-            doc="Save as Nist Qxy format."
-            "Note that if file formats of the same type, e.g. .xml are chosen, then the "
-            "file format is appended to the file name.",
+            doc="Save as Nist Qxy format. " + FILE_FORMAT_DOC_SUFFIX,
         )
         self.declareProperty(
             "RKH",
             False,
             direction=Direction.Input,
-            doc="Save as RKH format."
-            "Note that if file formats of the same type, e.g. .xml are chosen, then the "
-            "file format is appended to the file name.",
+            doc="Save as RKH format. " + FILE_FORMAT_DOC_SUFFIX,
         )
         self.declareProperty(
             "CSV",
             False,
             direction=Direction.Input,
-            doc="Save as CSV format."
-            "Note that if file formats of the same type, e.g. .xml are chosen, then the "
-            "file format is appended to the file name.",
+            doc="Save as CSV format. " + FILE_FORMAT_DOC_SUFFIX,
         )
 
         self.setPropertyGroup("Nexus", "FileFormats")
         self.setPropertyGroup("CanSAS", "FileFormats")
         self.setPropertyGroup("NXCanSAS", "FileFormats")
```

Comment:
Not affecting anything, but could we rename this:
```suggestion
        self.setPropertyGroup("NXcanSAS", "FileFormats")
```